WMIAdventure/frontend/src/js/components/profile/organisms/ChangeDeckCard/ChangeDeckCard.js

```diff
@@ -167,12 +145,13 @@ class ChangeDeckCard extends React.Component {
                             this.state.allCards.map((card) => {
                                 return (
                                     <React.Fragment key={`card-${card.id}`}>
-                                        <Card id={card.id} name={card.name}
+                                        <UserCard id={card.id} name={card.name}
                                               subject={card.subject}
                                               tooltip={card.tooltip}
                                               image={card.image}
                                               searchInput={this.state.searchInput}
-                                              levels={card.levels}
+                                              level={card.level}
```

Comment:
Tak jak pisałem wyżej, sam obecny *level*. Z tego co teraz robiłem przydałyby mi się wszystkie levele, macie jakiś pomysł gdzie i jak je uzyskać? Chyba że tutaj jakoś je dodać jako props z tablicą wszystkich poziomów *levels*, ale nie wiem czy to nie będzie rakowe.

Reply:
Ja bym po prostu wziął kartę po ID z tych metod ze storage. Nie wszystko trzeba propsami przekazywać. https://github.com/emkarcinos/WMIAdventure/blob/d8051510c29fdad5f42fd1faec8430cac63862bd/WMIAdventure/frontend/src/js/storage/cards/cardStorage.js#L7